ballot_api/settings/heroku.py

```diff
@@ -8,13 +8,20 @@
 
 DEBUG = False
 
-ALLOWED_HOSTS = [
-    'caciviclab-ballot-api.herokuapp.com',
-]
+
+if 'ALLOWED_HOSTS' not in os.environ:
```

Comment:
Try this:
```
ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', ['*'])
```
When it's stored in an env variable, we should be able to format like so: 
```export ALLOWED_HOSTS="['host.com', 'herokuapp.com', '*.localhost']"```